bindings/rust/evmc-vm/src/lib.rs

```diff
@@ -39,6 +39,14 @@ impl ExecutionResult {
         }
     }
 
+    pub fn failure() -> Self {
```

Comment:
Can you document these?

Reply:
Can you please?

Reply:
There's a lot to document. I suggest we create a bigger PR which documents the entire file.

Reply:
Created issue #308.